### ISSUE TITLE
spinner.gif not working in Next13

### ISSUE THREAD

**Issue by daenamcclintock** [aws-samples/amazon-cognito-passwordless-auth#41]
When trying to integrate this library into our frontend built with Next13, the spinner.gif included in the package is unable to resolve. I had to clone the repo and redeploy a new npm package without the spinner.gif in order for it to run without errors.

**Comment by ottokruse** [comment]
Thanks for the report. Can you share any more specifics around the error? And is it possible to provide steps/code to reproduce?

**Comment by ottokruse** [comment]
I was able to reproduce this by creating a sample next app and import the Passwordless React hooks and components (in a next component marked as `'use client'`, otherwise you run into another issue with `localStorage` being undefined).

Turns out when you do this in your JS code: `import spinner from "../assets/spinner.gif"`, as we do, NextJS wants you to access the source location using the `src` property, e.g. `<img src={spinner.src} />`, however we do `<img src={spinner} />` (without `.src`). Must be a bundling issue, Vite is fine with doing it as we do, and I believe it is the common way in React.

Note Next would support what we do **_if_** you use their `Image` component, instead of plain `img`: https://nextjs.org/docs/pages/building-your-application/optimizing/images

Not sure what to do with this. Any idea @EricBorland ?

**Comment by EricBorland** [comment]
Would the following work in Next?
```javascript 
import spinner from "../assets/spinner.gif"
const spinnerWrapper = {
  src: spinner
};

[...]
<img src={spinnerWrapper.src} />
```

If not, I guess another option is to reference the image by its url instead of importing directly in the code...

**Comment by ottokruse** [comment]
Fixed by #63 which is included in https://github.com/aws-samples/amazon-cognito-passwordless-auth/releases/tag/v0.6.6 onwards
